pkgs/tools/text/xpaste/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "xpaste";
+  version = "1.5";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ossobv";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Z9YxhVP+FUvWET91ob4SPSW+XX7/wzlgPr53vO517L4=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    xlib
+  ];
+
+  # no tests
+  doCheck = false;
```

Comment:
Can we do an pythonImportsCheck instead?

Reply:
No, upstream does not provide a module which could be imported.
`site-pakckages` does not contain a module in `out/lib/python3.10/site-packages`:

```
ls -la result/lib/python3.10/site-packages
dr-xr-xr-x - root  1 Jan  1970 xpaste-1.5.dist-info
```